composer.json

```diff
@@ -46,13 +46,12 @@
 		"phpstan/phpstan-strict-rules": "^1.5.1",
 		"phpunit/phpunit": "^9.6.7",
 		"saschaegerer/phpstan-typo3": "^1.8.4",
-		"sbuerk/typo3-cmscomposerinstallers-testingframework-bridge": "^0.1.3",
 		"seld/jsonlint": "^1.9.0",
 		"squizlabs/php_codesniffer": "^3.7.2",
 		"symfony/yaml": "^5.4 || ^6.1",
 		"typo3/cms-fluid-styled-content": "^11.5.4 || ^12.4",
 		"typo3/coding-standards": "^0.6.1",
-		"typo3/testing-framework": "^7.0@dev"
+		"typo3/testing-framework": "^7.0.0"
```

Comment:
Out of curiosity: Why `^7.0.0` and not just `^7.0` (like for the other the requirements)?

Reply:
I'd like to also control the bug-fix release version of the dev dependencies. (And I think I'd like to change the other dev dependencies accordingly except for `symfony/yaml`, which also is an (indirect) production dependency and which we'd hence also like to check with the lowest/highest dependency versions.)

Reply:
Okay, but `^7.0` and `^7.0.0` are equivalent. But fine for me.